ega-data-api-dataedge/src/main/java/eu/elixir/ega/ebi/shared/service/internal/AuthenticationServiceImpl.java

```diff
@@ -33,4 +38,24 @@ public String getName() {
     return null;
   }
 
+    @Override
+    public String getDatasetIdByStableId(String stableId) {
```

Comment:
Actually I was thinking that the Authentication Service returns the datasetFileMapping and then the Permissions service is the one that matches the stable id to the datasetfilemapping. So the authentication services gives the information but the permission service is the ones that knows what to do with it. Does it makes sense? 
Sorry for being so picky.

Reply:
Yes, this is better for separating concerns. Updated the code.